scripts/changelog-ci.py

```diff
@@ -65,12 +69,18 @@ def _get_pull_request_title_and_number(event_path):
 
         return title, number
 
-    def _parse_config(self, config_file):
+    def _parse_config(self, filepath: str):
         """parse the config file if not provided use default config"""
-        if config_file:
+        if filepath:
             try:
-                with open(config_file, 'r') as config_json:
-                    config = json.load(config_json)
+                file = open(filepath)
```

Comment:
Don't we need to close the file?

Reply:
Yes, the garbage collector will eventually close the file, but Isn't it good practice to always explicitly close the file without relying on the garbage collector?

Reply:
I agree it is good practice to do so, but my response was answering your first question:

> Don't we need to close the file?

Reply:
Oh, Alright 👍 The PR is almost ready to get merged 💯, I think we should close the opened file here 🙂

Reply:
Go for it! You have write access to this branch

Reply:
Sure, I'm merging this to another branch before merging to master as I need to test the YAML config as well, thanks for your great work on this :)